[2023-10-17T11:19:18.073+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-29T00:00:00+00:00 [queued]>
[2023-10-17T11:19:18.082+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-29T00:00:00+00:00 [queued]>
[2023-10-17T11:19:18.083+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:18.138+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-29 00:00:00+00:00
[2023-10-17T11:19:18.145+0000] {standard_task_runner.py:57} INFO - Started process 609 to run task
[2023-10-17T11:19:18.148+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-29T00:00:00+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpg9dopzhq']
[2023-10-17T11:19:18.148+0000] {standard_task_runner.py:85} INFO - Job 92: Subtask waiting_for_data
[2023-10-17T11:19:18.193+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-29T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:18.279+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-29T00:00:00+00:00'
[2023-10-17T11:19:18.288+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:18.289+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:18.289+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:18.289+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:18.297+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210329T000000, start_date=20231017T111918, end_date=20231017T111918
[2023-10-17T11:19:18.320+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:18.332+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:42:47.768+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-29T00:00:00+00:00 [queued]>
[2023-10-18T10:42:47.781+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-29T00:00:00+00:00 [queued]>
[2023-10-18T10:42:47.781+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:42:47.801+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-29 00:00:00+00:00
[2023-10-18T10:42:47.811+0000] {standard_task_runner.py:57} INFO - Started process 1573 to run task
[2023-10-18T10:42:47.816+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-29T00:00:00+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpnn_n3b41']
[2023-10-18T10:42:47.818+0000] {standard_task_runner.py:85} INFO - Job 91: Subtask waiting_for_data
[2023-10-18T10:42:47.873+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-29T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:42:47.939+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-29T00:00:00+00:00'
[2023-10-18T10:42:47.962+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:42:47.967+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210329T000000, start_date=20231018T104247, end_date=20231018T104247
[2023-10-18T10:42:47.983+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 91 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1573)
[2023-10-18T10:42:48.032+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:42:48.098+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:19.953+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-29T00:00:00+00:00 [queued]>
[2023-10-18T11:17:19.966+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-29T00:00:00+00:00 [queued]>
[2023-10-18T11:17:19.966+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:19.983+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-29 00:00:00+00:00
[2023-10-18T11:17:19.992+0000] {standard_task_runner.py:57} INFO - Started process 402 to run task
[2023-10-18T11:17:19.996+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-29T00:00:00+00:00', '--job-id', '135', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpao4bu7xw']
[2023-10-18T11:17:19.997+0000] {standard_task_runner.py:85} INFO - Job 135: Subtask waiting_for_data
[2023-10-18T11:17:20.061+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-29T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:20.156+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-29T00:00:00+00:00'
[2023-10-18T11:17:20.171+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:20.174+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:20.175+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:20.175+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:20.188+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210329T000000, start_date=20231018T111719, end_date=20231018T111720
[2023-10-18T11:17:20.220+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:20.270+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:11:59.876+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-29T00:00:00+00:00 [queued]>
[2023-10-19T12:11:59.886+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-29T00:00:00+00:00 [queued]>
[2023-10-19T12:11:59.886+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:11:59.903+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-29 00:00:00+00:00
[2023-10-19T12:11:59.911+0000] {standard_task_runner.py:57} INFO - Started process 458 to run task
[2023-10-19T12:11:59.915+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-29T00:00:00+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpr1pv2bs8']
[2023-10-19T12:11:59.917+0000] {standard_task_runner.py:85} INFO - Job 136: Subtask waiting_for_data
[2023-10-19T12:11:59.968+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-29T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:12:00.049+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-29T00:00:00+00:00'
[2023-10-19T12:12:00.062+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:12:00.062+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:12:00.062+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:12:00.063+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:12:00.074+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210329T000000, start_date=20231019T121159, end_date=20231019T121200
[2023-10-19T12:12:00.129+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:12:00.173+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
